app/models/public_body.rb

```diff
@@ -364,10 +375,12 @@ def site_administration?
 
   # Can an FOI (etc.) request be made to this body?
   def is_requestable?
-    has_request_email? && !defunct? && !not_apply?
+    has_request_email? && !defunct? && !not_apply? && !not_requestable?
   end
 
-  scope :is_requestable, -> { with_request_email.not_defunct.foi_applies }
+  scope :is_requestable, -> {
+    with_request_email.not_defunct.foi_applies.requestable
```

Comment:
`is_requestable` and `requestable` scopes feel quite prone to confusion. I'd be inclined to go with:

```rb
with_request_email.not_defunct.foi_applies.without_tag('not_requestable')
```